.github/workflows/cmake-configure.yml

```diff
@@ -16,6 +16,9 @@ jobs:
             cuda-mode: [AUTO, Clang-CUDA, NVCC-CUDA]
     steps:
     - uses: actions/checkout@v2
+    - name: Install dependencies
```

Comment:
with the soft failure we can remove that again

Reply:
Removed numpy from CI.